handbook/product/roadmap.md

```diff
@@ -28,7 +28,14 @@ section Cloud
 %% section Code intel
 %%     TO DO :active, after release-3.21, 30d
 
-%% section Web
+section Web
+    Browser extension discoverability                         :active, 2020-09-28, 14d
+    Build new and improved extensions                         :         2020-10-12, 14d
```

Comment:
```suggestion
    Browser extension discoverability                         :               2020-09-28, 14d
    Build new and improved extensions                         :active         2020-10-12, 14d
```

Reply:
there's actually a `:done` status I need to add, so making that update along with this one!